Directory.Build.props

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Project>
   <PropertyGroup>
-    <NetVersion>net8.0</NetVersion>
-    <MauiPackageVersion>8.0.71</MauiPackageVersion>
+    <NetVersion>net9.0</NetVersion>
+    <MauiPackageVersion>9.0.0-rc.2.24503.2</MauiPackageVersion>
```

Comment:
Reminder for us to change this to the stable release version of `Microsoft.Maui.Controls` before merging

Reply:
```suggestion
    <MauiPackageVersion>9.0.0</MauiPackageVersion>
```